utils/hct/hctdb_inst_docs.txt

```diff
@@ -588,3 +588,65 @@ dest0, dest1 = USubb(src0, src1)
 * Inst: AttributeAtVertex - returns the values of the attributes at the vertex.
 
 returns the values of the attributes at the vertex. VertexID ranges from 0 to 2.
+
+* Inst: FDiv - returns the quotient of its two operands
+
+%des = fdiv float %src0, %src1
+
+The following table shows the results obtained when executing the instruction with various classes of numbers, assuming that neither overflow or underflow occurs.
+
+One outcome of this is there are exceptions to the table below for large denominator values (greater than 8.5070592e+37), where 1/denominator is a denorm. Since implementations may perform divide as a*(1/b), instead of a/b directly, and 1/[large value] is a denorm that could get flushed, some cases in the table would produce different results. For example (+/-)INF / (+/-)[value > 8.5070592e+37] may produce NaN on some implementations, but (+/-)INF on other implementations.
+For denorms, the table reflects "fp32-denorm-mode"="preserve" case. for "fp32-denorm-mode"="ftz" mode, denorms are treated as zeros.
```

Comment:
It looks like we already have a paragraph talking about how implementation may use a*(1/b) here.  Can we simplify descriptions to something like:
```
The following table shows the results obtained when executing the instruction with various classes of numbers, assuming that fast math flag is not used and "fp32-denorm-mode"="preserve".
When "fp32-denorm-mode"="ftz", denorm inputs should be interpreted as corresponding signed zero, and any resulting denorm is also flushed to zero.
When fast math is enabled, implementation may use reciprocal form: a*(1/b).  This may result in evaluating a*(+/-)INF from a*(1/(+/-)denorm).  This may produce NaN in some cases or (+/-)INF in others.
```

Reply:
Update to last line of text:
`When fast math is enabled, implementation may use reciprocal form: a*(1/b).  This may result in evaluating a*(+/-)NaN from a*(1/(+/-)INF), or a*(+/-)INF from a*(1/(+/-)denorm).  This may produce NaN in some cases, or (+/-)INF in others.`